test/test-style.js

```diff
@@ -136,23 +136,23 @@ function processData(filename, logger) {
   /** @type {import('../types').CompatData} */
   let dataObject = JSON.parse(actual);
   let expected = JSON.stringify(dataObject, null, 2);
-  let expectedSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
+  let expectedBrowserSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
 
   // prevent false positives from git.core.autocrlf on Windows
   if (IS_WINDOWS) {
     actual = actual.replace(/\r/g, '');
     expected = expected.replace(/\r/g, '');
-    expectedSorting = expectedSorting.replace(/\r/g, '');
+    expectedBrowserSorting = expectedBrowserSorting.replace(/\r/g, '');
   }
 
   if (actual !== expected) {
     hasErrors = true;
     logger.error(chalk`{red Error on {bold line ${jsonDiff(actual, expected)}}}`);
   }
 
-  if (expected !== expectedSorting) {
+  if (expected !== expectedBrowserSorting) {
     hasErrors = true;
-    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedSorting)}}}`);
+    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedSorting)}}}\n{blue     Tip: Run {bold npm run fix} to fix sorting automatically}`);
```

Comment:
Missing one change here (also the fix script isn't in yet, so alternatively you can drop the tip too and restore it when it gets added)

```suggestion
    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedBrowserSorting)}}}\n{blue     Tip: Run {bold npm run fix} to fix sorting automatically}`);
```

```suggestion
    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedBrowserSorting)}}});
```

Reply:
WHOOPS!  Let’s just slip this one under the rug, shall we?

Since the fix script isn’t around, we might as well remove the suggestion until it’s in!